.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+#### Do you want to request a *feature* or report a *bug*?
+
+#### What's the current behavior?
+
+<!--
+For bugs, please include the following if possible:
+
+  - A CodeSandbox that minimally reproduces the issue.
+  - An image or GIF showing the issue.
+  - Information about your browser, react-big-calendar version, etc.
+
+http://recordit.co/
+https://codesandbox.io/s/l9jwl0kj6m
+
+If you don't include these, there's a very good chance your issue will be closed, because it's too hard to figure out exactly what is going wrong, and it makes maintenance much harder.
```

Comment:
I might soften this a bit, but otherwise 👍

Reply:
👍 I just removed that paragraph...